src/cdk/tree/tree.ts

```diff
@@ -255,6 +256,8 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
       },
     );
 
+    this._changeDetectorRef.markForCheck();
```

Comment:
@atscott does this look like the right solution here?

Reply:
Hmm, it's hard to say. This component is pretty complicated and I don't have enough background. I don't know why this would have been calling `detectChanges` instead of `markForCheck`.

Reply:
Looks like I don't actually need to make this change after fixing the host bindings, but leaving a TODO anyways, because we should clean up the call to `detectChanges`